core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2229,6 +2235,8 @@ public String toString() {
         "alluxio.master.journal.flush.batch.time";
     public static final String MASTER_JOURNAL_FLUSH_TIMEOUT_MS =
         "alluxio.master.journal.flush.timeout";
+    public static final String MASTER_JOURNAL_FLUSH_RETRY_INTERVAL =
+        "alluxio.master.journal.retry.interval";
```

Comment:
When do we expect this value to be set by the user?

Reply:
never, it is hidden. It's added as a property key for testing and debugging purposes.

Reply:
Is there an alternative mechanism in master for exposing a property for testing/debugging only?